회칙.md

```
@@ -2,73 +2,83 @@
 
 # SCSC 회칙
 
-> 작성: 컴퓨터공학부 강명석  
-> 개정: 컴퓨터공학부 강명석  
-> 최신개정일 25-07-27  
+> 최초 작성일: 2025-08-17
+>
+> 최신 개정일: 2025-08-22
+>
+> 최신 개정자: [오현우CSE]()
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)